packages/wrangler/templates/startDevWorker/ProxyWorker.ts

```diff
@@ -129,7 +129,6 @@ export class ProxyWorker implements DurableObject {
 				request.url
 			);
 			headers.set("MF-Original-URL", innerUrl.href);
-			headers.set("MF-Disable-Pretty-Error", "true"); // disables the UserWorker miniflare instance from rendering the pretty error -- instead the ProxyWorker miniflare instance will intercept the json error response and render the pretty error page
```

Comment:
We are now relying on the UserWorker miniflare instance to prettify the error so that we can retrieve the script source from its inspector.